src/utils/components/SSHAccess/utils.ts

```diff
@@ -106,5 +107,14 @@ export const createSSHService = async (
   });
 };
 
-export const getConsoleVirtctlCommand = (userName: string, vmName: string, vmNamespace?: string) =>
-  `virtctl ${vmNamespace ? `-n ${vmNamespace}` : ''} ssh ${userName}@${vmName}`;
+export const getConsoleVirtctlCommand = (vm: V1VirtualMachine, identityFlag?: string) => {
+  const [vmName, vmNamespace, userName] = [
+    getName(vm),
+    getNamespace(vm),
+    getCloudInitCredentials(vm)?.users?.[0]?.name,
+  ];
+
+  return `virtctl -n ${vmNamespace} ssh ${userName}@${vmName} ${
+    !isEmpty(identityFlag) ? identityFlag : '[--identity-file]'
```

Comment:
I do not think you should put the flag in square brackets because that implies that the flag is optional. When I tested the 'virtctl ssh' commands, I was unable to connect without specifying the '--identity-file' flag. The VM simply could not find the right key without that flag, even if the key was in the default location (/home/user/.ssh). I recommend you change it to something like '--identity-file=<ssh_key>'.